plan/plans/perf_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plans_test
+
+import (
+	"database/sql"
+
+	. "github.com/pingcap/check"
+	"github.com/pingcap/tidb"
+)
+
+type testPerfSchemaSuit struct {
+	vars map[string]interface{}
+}
+
+var _ = Suite(&testPerfSchemaSuit{
+	vars: make(map[string]interface{}),
+})
+
+func (p *testPerfSchemaSuit) TestPerfSchema(c *C) {
+	testDB, err := sql.Open(tidb.DriverName, tidb.EngineGoLevelDBMemory+"/test/test")
+	c.Assert(err, IsNil)
+	cnt := mustQuery(c, testDB, "select * from performance_schema.setup_actors")
```

Comment:
It is better to add some records then test query, now seems empty record returned.
But maybe next PR.